src/lua/lua_script.rs

```diff
@@ -141,31 +148,23 @@ impl LuaScript {
     }
 
     pub fn reset(&mut self, dimensions: (u16, u16)) {
-        let store = self
-            .state
-            .context(|ctx| -> Result<Store, mlua::Error> {
-                ctx.globals().get(Store::LUA_GLOBAL_NAME)
-            })
-            .ok();
+        let store = self.state.globals().get(Store::LUA_GLOBAL_NAME).ok();
         self.state = create_default_lua_state(self.writer.clone(), dimensions, store);
     }
 
     pub fn get_output_lines(&self) -> Vec<Line> {
-        self.state
-            .context(|ctx| -> LuaResult<Vec<Line>> {
-                let blight_aud: AnyUserData = ctx.globals().get("blight")?;
-                let mut blight = blight_aud.borrow_mut::<Blight>()?;
-                let lines = blight.get_output_lines();
-                Ok(lines)
-            })
-            .unwrap()
+        let blight_aud: AnyUserData = self.state.globals().get("blight").unwrap();
+        let mut blight = blight_aud.borrow_mut::<Blight>().unwrap();
```

Comment:
I'd prefer `.expect("Some message about bad 'blight' object")` since we are trying to reduce the unwrap count. Expects makes it more clear that this a valid hard-crash case.